src/renderer.zig

```diff
@@ -752,18 +752,9 @@ pub const Skybox = struct {
 		skyShader = Shader.initAndGetUniforms("assets/cubyz/shaders/skybox/sky.vs", "assets/cubyz/shaders/skybox/sky.fs", "", &skyUniforms);
 		skyShader.bind();
 
-		const rawData = [_]f32 {
-			-1, -1, -1,
-			1, -1, -1,
-			1, 1, -1,
-			-1, 1, -1,
-			-1, -1, 1,
-			1, -1, 1,
-			1, 1, 1,
-			-1, 1, 1
```

Comment:
I think it should be kept in separate lines, you can tell the formatter to keep it if you add a trailing comma after the last element.